More2Life/BuyModalViewController.swift

```diff
@@ -77,7 +79,7 @@ class BuyModalViewController: UIViewController, ApplePaying {
             addApplePayButton(with: .donate)
             actionButton?.setTitle(localizedDonateString, for: .normal)
         case .action(let feedItem):
-            productNameLabel.text = feedItem.title
+			productNameLabel.text = (feedItem.title ?? "Selection") + ":"
```

Comment:
Use a string here. Concatenation takes a long time to compile for some reason:

`"\(feedItem.title ?? "Selection"):"`